### ISSUE TITLE
How can i run so that i can get the map as your list？

### ISSUE THREAD

**Issue by Hua2544** [ultralytics/ultralytics#17846]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

sorry，can you tell me your parms running in the coco-pose dataset？i cant get your  map  with  default parms.  or  whats wrong with my experments?

### Additional

sorry，can you tell me your parms running in the coco-pose dataset？i cant get your  map  with  default parms.  or  whats wrong with my experments?

**Comment by UltralyticsAssistant** [comment]
👋 Hello @Hua2544, thank you for your interest in Ultralytics 🚀! We recommend a visit to the [Docs](https://docs.ultralytics.com) for new users, where you can find many [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples and where many of the most common questions may already be answered.

If this is a ❓ Question regarding training parameters and results, please provide as much information as possible to help us assist you. This includes:

1. The exact command or code you are using to run the experiments.
2. Any training logs or outputs from your experiments.
3. Details about your environment, such as hardware (e.g., GPU specs) and software versions.
4. Any modifications made to the default Ultralytics YOLO configuration or hyperparameters.

If this is related to a 🐛 Bug in the code, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) so we can debug it effectively.

We also highly recommend checking out our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/) to ensure you're following best practices when running experiments.

## Upgrade

Ensure you're using the latest version of the `ultralytics` package along with its dependencies. Upgrade in a [**Python>=3.8**](https://www.python.org/) environment using:

```bash
pip install -U ultralytics
```

This will ensure that you have the latest improvements and bug fixes.

## Environments

YOLO can be run in any of the following environments pre-configured with dependencies ([CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/), and [PyTorch](https://pytorch.org/)):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Community Support

Join the Ultralytics community where it suits you best:
- Real-time chat? Join us on [Discord](https://discord.com/invite/ultralytics) 🎧
- In-depth discussions? Visit [Discourse](https://community.ultralytics.com) 💬
- Share ideas and learn on our [Subreddit](https://reddit.com/r/Ultralytics) 📚

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

This is an automated response to assist you quickly, but don't worry—a dedicated Ultralytics engineer will respond soon to address your question directly 😊.

**Comment by Y-T-G** [comment]
You can get it from the model

https://github.com/ultralytics/ultralytics/issues/17048#issuecomment-2424742030

**Comment by RunenrCode** [comment]
![Uploading tt.png…]()

**Comment by Hua2544** [comment]
thanks,
model = YOLO("yolov8n-pose.pt")
print(model.ckpt["train_args"]),can i also get  model args  ?   

> You can get it from the model
> 
> [#17048 (comment)](https://github.com/ultralytics/ultralytics/issues/17048#issuecomment-2424742030)

**Comment by Hua2544** [comment]
> You can get it from the model
> 
> [#17048 (comment)](https://github.com/ultralytics/ultralytics/issues/17048#issuecomment-2424742030)

i get other wrong,some args will not be in yolo,
![image](https://github.com/user-attachments/assets/4e8a9d8d-a69c-47bd-8c37-84d6cf222d29)
![image](https://github.com/user-attachments/assets/2fa12703-217f-433c-b209-48dd0372a805)
so,i cant reapte the yolov8-pose with coco-pose

**Comment by Skillnoob** [comment]
Those are the hyperparameters, there's nothing else to change

**Comment by Hua2544** [comment]
> Those are the hyperparameters, there's nothing else to change
i need the same parms to get the same map,but i cant

**Comment by RunenrCode** [comment]
This is an error message, using my dataset
[https://github.com/RunenrCode/imgdataset-test/issues/1](url)

**Comment by Skillnoob** [comment]
@RunenrCode create a new issue, don't use another issue for a different problem.

**Comment by Skillnoob** [comment]
@Hua2544 what ultralytics version are you using?
Please update with `pip install ultralytics -U` and try again

**Comment by Hua2544** [comment]
will,it doesnt work , i only make cache=False,and others as same as yolov8-pose.what i get is only 0,the all loss or other pre is zero.

> @Hua2544 what ultralytics version are you using? Please update with `pip install ultralytics -U` and try again

thank you for your reply!

**Comment by Hua2544** [comment]
the version maybe too old,but it matters nothing,i think

**Comment by Y-T-G** [comment]
Can you provide the output after running this command in terminal: `yolo checks`?

**Comment by Hua2544** [comment]
> Can you provide the output after running this command in terminal: `yolo checks`?

whats my problem ? thank you for your reply
!

**Comment by Hua2544** [comment]
> Can you provide the output after running this command in terminal: `yolo checks`?

![image](https://github.com/user-attachments/assets/b6b983dd-821b-4909-b5a6-d35246a37455)

**Comment by Y-T-G** [comment]
> > Can you provide the output after running this command in terminal: `yolo checks`?
> 
> ![image](https://private-user-images.githubusercontent.com/123644107/391061522-bb18a019-618d-440a-8e5c-95765507ccf9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI4Nzk0MDIsIm5iZiI6MTczMjg3OTEwMiwicGF0aCI6Ii8xMjM2NDQxMDcvMzkxMDYxNTIyLWJiMThhMDE5LTYxOGQtNDQwYS04ZTVjLTk1NzY1NTA3Y2NmOS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEyOVQxMTE4MjJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03ZmIwNWU1NDJmOTA3YmI3M2M5YzFhMGE0OTE0ZjQzYjE2ZmMwOWZkYThkZTQyYzMwODkwZDlmMTYwNzZkYTcwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.G4Vhh0dO8ebEQE_oNdrWAfe50HWc8qr-hrgqzskN5G8)

That's not the correct output. Run `yolo checks` in your terminal/shell/command line/command prompt

**Comment by Skillnoob** [comment]
> @Hua2544 what ultralytics version are you using? Please update with `pip install ultralytics -U` and try again

@Hua2544 I already told you to upgrade your ultralytics version, please do it.
